summarizer/summarizer.py

```
@@ -0,0 +1,292 @@
+import discord
+from redbot.core import commands, Config, app_commands
```

Ruff (0.8.2)

`redbot.core.app_commands` imported but unused

Remove unused import: `redbot.core.app_commands`

(F401)